# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1321823975 28800
#      Sun Nov 20 13:19:35 2011 -0800
# Node ID 2f73cbd951fb4422a277fa1845f8888ff5900f50
# Parent  d4404589498c1ef717f585bd8029b0a33a3b4424
doc: Add Vanya Sergeev to list of contributors

* contributors.in: Add Vanya Sergeev to list of contributors

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -205,16 +205,17 @@ Alois Schloegl
 Michel D. Schmid
 Julian Schnidder
 Nicol N. Schraudolph
 Sebastian Schubert
 Ludwig Schwardt
 Thomas L. Scofield
 Daniel J. Sebald
 Dmitri A. Sergatskov
+Vanya Sergeev
 Baylis Shanks
 Andriy Shinkarchuck
 Joseph P. Skudlarek
 John Smith
 Julius Smith
 Shan G. Smith
 Joerg Specht
 Quentin H. Spencer
